docs/openapi.yml

```diff
@@ -88,6 +157,8 @@ paths:
           description: Limits the number of items to return.
           schema:
             type: integer
+            minimum: 1
```

Comment:
0 should be allowed I think

Reply:
this is for `top-k`? what does it mean to `query.limit(0)` here?

Reply:
it just means you'll get 0 results back. It works in SQL :man_shrugging:

Reply:
lets do it until we have a fast path?